package.json

```diff
@@ -18,7 +18,7 @@
     "ast-types": "git+https://github.com/jlongster/ast-types.git",
     "babylon": "^6.15.0",
     "esutils": "^2.0.2",
-    "flow-parser": "^0.37.0",
+    "flow-parser": "0.38.0",
```

Comment:
`yarn add` doing. I actually think that `^` behavior is really bad but I'll revert :)